index.html

```diff
@@ -13739,6 +13739,10 @@ <h2>Changes since Candidate Release of 12 December 2019</h2>
       and `rdf:CompoundLiteral` class in <a href="#the-rdf-compoundliteral-class-and-the-rdf-language-and-rdf-direction-properties" class="sectionRef"></a>
       to normalize language tags to lowercase when generating RDF.</li>
     <li>Update bibliographic reference for JCS to [[RFC8785]].</li>
+    <li>Fixed typo in <a href="#frame-objects" class="sectionRef"></a>,
```

Comment:
This entry should go into a separate appendix, with the changes since Proposed Recommendation Release of May 7 2020

Reply:
oops, sorry about that. fixed in latest commit